CODEOWNERS

```diff
@@ -4,6 +4,10 @@
 # These names are just suggestions. It is fine to have your changes
 # reviewed by someone else.
 
+app_dart                                           @keyonghan
+app_dart                                           @ricardoamador
+auto_submit                                        @keyonghan
+auto_submit                                        @ricardoamador
```

Comment:
Can we just use the infra team for this instead of people directly?

Reply:
Happy to! What group is it? I don't see it in https://github.com/orgs/flutter/teams

Reply:
i am a bit reluctant to add the team here. The ownership here will trigger review request for any changes happened from the Dir.
@ricardoamador Are you okay to own the whole auto_submit dir? If not, let's divide it into smaller pieces. Same for the app_dart.
The reason I put the ownership on dir level for cipd packages is they are relatively a small self-contained process. As for app_dart, it would be too large to give a single ownership entry on the dir level. But as for `auto_submit`, up to you @ricardoamador

Reply:
I will defer to Keyong.

Reply:
As for auto_submit, I prefer to keep only @ricardoamador (assuming okay:-) as the owner considering he has been the main contact/contributor of this app and the app is not that big. 
As for app_dart, let's skip it for now. We want to break it into smaller pieces on API/file level.

Reply:
I would go the opposite way. Can we add the entire engprod team as owners? Having a single person be the owner creates a bottleneck, and the best practice is to ensure owners have redundancy.

Reply:
CODEOWNERS file here is behaving differently from Google. It just auto adds the owner as a code reviewer, and will not be a bottleneck. People can add anybody else as a code reviewer.

Reply:
Done. Applied feedback for now to only add the autosubmit CODEOWNERS